components/Shell.tsx

```diff
@@ -76,6 +76,8 @@ function useRedirectToOnboardingIfNeeded() {
   useEffect(() => {
     if (user && shouldShowOnboarding(user)) {
```

Comment:
This was the culprit, shouldShowOnboarding(user) was returning true because user.completedOnboarding hasn't been set to true when the redirect occurs, and nothing was setting isRedirecting back to false.

Which is confusing because the redirect from getting-started to event-types doesn't occur until after updateUser has been run to set completedOnboarding to true..

This fixes it by setting isRedirecting back to false, but i feel like there should be a better way to do it than this

Reply:
A better way might be changing something in getting-started.tsx lines 209-215 to get the redirect to wait longer for updateUser(); but i'm not sure what that would look like